### PR TITLE
Add nushell to enable-tab-autocomplete.md

### LINKED ISSUE #34944: Add nushell to enable-tab-autocomplete.md
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 341f0255-aa31-8e76-9b17-1232b7a910a9
* Version Independent ID: f30f63dc-0d90-7dba-ef39-b0148c637c1a
* Content: [Enable tab completion - .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tools/enable-tab-autocomplete?branch=pr-en-us-34942#nushell)
* Content Source: [docs/core/tools/enable-tab-autocomplete.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/enable-tab-autocomplete.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

---
[Associated WorkItem - 84627](https://dev.azure.com/msft-skilling/Content/_workitems/edit/84627)

### PR DESCRIPTION
**Pull request by ShinyZero0**
## Summary
Add instructions for enabling completion of .NET CLI commands in [nushell](https://github.com/nushell/nushell)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tools/enable-tab-autocomplete.md](https://github.com/dotnet/docs/blob/a0f7074d24b7774f374b8e5238d971d990953eb6/docs/core/tools/enable-tab-autocomplete.md) | [zsh parameter completion for the dotnet CLI](https://review.learn.microsoft.com/en-us/dotnet/core/tools/enable-tab-autocomplete?branch=pr-en-us-34942) |

_tdykstra edit to add link to issue_

Fixes #34944

### CODE REVIEW

**Comment by ShinyZero0** [comment]
@dotnet-policy-service agree

File: docs/core/tools/enable-tab-autocomplete.md [hunk 1]
```diff
@@ -106,3 +106,18 @@ To add tab completion to your **fish** shell for the .NET CLI, add the following
 ```fish
 complete -f -c dotnet -a "(dotnet complete (commandline -cp))"
 ```
+
+## nushell
+
+To add tab completion to your **nushell** for .NET CLI, add the following to your `config.nu` file:
+```nu
```
Comment (tdykstra) [inline docs/core/tools/enable-tab-autocomplete.md:113]:
```suggestion
```nu

```

File: docs/core/tools/enable-tab-autocomplete.md [hunk 2]
```diff
@@ -106,3 +106,19 @@ To add tab completion to your **fish** shell for the .NET CLI, add the following
...
+
+# completion for the .NET CLI
```
Comment (tdykstra) [inline docs/core/tools/enable-tab-autocomplete.md:115]:
```suggestion
To add tab completion to your **nushell** for .NET CLI, add the following to your `config.nu` file:

```nu
# completion for the .NET CLI
```

**Review by tdykstra — APPROVED** [review]
Thanks, @ShinyZero0 !

**Comment by tdykstra** [comment]
Close and reopen to restart doc build

**Review by adegeo — APPROVED** [review]
Verified as working

**Pull request merged**